pinc/access_change_emails.inc

```diff
@@ -56,6 +56,95 @@ function pp_access_change_email($user, $access_change)
+
+    // only send emails on grant
+    if ($access_change != "grant") {
+        return;
+    }
+
+    // configure gettext to translate user email
+    configure_gettext_for_user($user);
+
+    // TRANSLATORS: %s is the site abbreviation (eg: 'DP')
+    $subject = sprintf(_("%s: Welcome to P2 Proofreading"), $site_abbreviation);
+
+    $message[] = sprintf(_("Hello %s,"), $user->username);
+
+    $message[] = _("Congratulations! You may now work in P2, the second proofreading round.");
+
+    $message[] = "== " . _("Compared to P1, what remains the same?") . " ==";
+
+    $message[] = _("The Proofreading Guidelines are exactly the same, and you still need to comply with the Project Comments and Project Discussion guidance for every project.");
+
+    $message[] = _("Please keep aiming to leave every page as perfect as you can.");
+
+    $message[] = "== " . _("What is expected in P2?") . " ==";
+
+    $message[] = _("Using WordCheck is compulsory in P2 and P3, and pages that have not been WordChecked may be removed from your record.");
+
+    $message[] = _("It is important to be aware that, when you work in P2, you set an example to P1 proofreaders: they will be checking their diffs to see how you changed their work. Make sure that you don't mislead them.");
+
+    $message[] = _("If you remember how your mentor guided you when you started, you may consider giving gentle feedback in a private message to new proofreaders when you see them making repeated errors.");
+
+    $message[] = _("The text you worked on in P1 was generated from the page images by means of software tools. When you work in P2, however, you are working on text that someone else has already proofread. Nevertheless, please check every character and word carefully, since new P1s, unfamiliar with our guidelines, may have left errors.");
+
+    if ($daily_limit) {
+        $message[] = sprintf(_("Please keep in mind that we currently have a limit in P2 of no more than %d pages per day. This is a daily limit, not an average; the limit is there to make sure that we have a variety of projects available in the round at all times."), $daily_limit);
+        // The above paragraph will be added to the email only if the daily limit
+        // is a positive integer. Currently, if the daily limit value is zero or
+        // NULL, the paragraph is not added. This code will need to be adjusted
+        // if there is ever a need for them to be treated differently.
+    }
+
+    $message[] = "== " . _("What resources are available?") . " ==";
+
+    $message[] = "- " . sprintf(_("You may ask for help at dp-feedback ( %s ), giving the name of the project, the round you worked in, and the png numbers of the pages you would like to have reviewed."), "https://www.pgdp.net/phpBB3/ucp.php?i=pm&mode=compose&u=37603");
+
+    $message[] = "- " . sprintf(_("If you haven't yet, try using DP’s Proofreading Font: %s"), "https://www.pgdp.net/c/faq/font_sample.php");
```

Comment:
we use straight apostrophe in other strings (...don't mislead...) and even this one, but here an actual apostrophe. Wes hould be consistent I imagine.

Reply:
Double-check me, please, but I think I got that fixed.

Reply:
I think it is good to me.